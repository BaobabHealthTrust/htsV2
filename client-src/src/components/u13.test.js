import React from 'react';
import ReactDOM from 'react-dom';
import U13 from './u13';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import Enzyme, {
    shallow
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
});

describe('Component: U13', () => {

    it('should match its empty snapshot', () => {
        const tree = renderer.create( 
            <U13 />
        ).toJSON();

        expect(tree).toMatchSnapshot();
    })

})