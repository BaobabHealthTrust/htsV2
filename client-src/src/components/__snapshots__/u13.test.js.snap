// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: U13 should match its empty snapshot 1`] = `
<div
  className="nav"
/>
`;
